### ISSUE TITLE
Update HelperConfigTest once zend-view refactor is complete

### ISSUE THREAD

**Issue by weierophinney** [zendframework/zend-navigation#6]
The HelperConfigTest test case is marked incomplete currently due to the fact that `Zend\View\Navigation\PluginManager` is not updated to the latest zend-servicemanager changes. Once it has been, we can address this test case.

**Comment by Maks3w** [comment]
I think the test should be refactored then for not depend of the external implementation.

**Comment by weierophinney** [comment]
The point of the test is to determine how it configures the view helper
manager; it's an *integration* test.

**Comment by weierophinney** [comment]
This repository has been closed and moved to laminas/laminas-navigation; a new issue has been opened at https://github.com/laminas/laminas-navigation/issues/4.
